p1pop.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="s1pop.css" type="text/css" rel="stylesheet">
+        <link href="css/fontello.css" type="text/css" rel="stylesheet">
+
+        <meta charset="Utf-8">
+    </head>
+    <body>
+        <main>
+               <div class="navbar">
+                <i class="demo-icon icon-search"></i>                
+                <ul>
+                    <li>About me</li>
+                    <li>Skils</li>
+                    <li>Interested me</li>
+                </ul>
+            </div>
+                <div class="pojemnik">
+                    <div class="Napis">
+                        <h1>Welcome!!</h1>
+                        <p>My name is designer.</p>
+                        <div class="Media">
+                            
+                        </div>
+                    </div>
+                    <img src="cover photo.png">
+                </div>
+                <div class="Pz">
+                    <div class="np">I am happy to help you with a problem.</div>
+                    <button>Contact with me!</button>
+                    <button>Read about me!</button>
+                </div>
+                <div class="Aboutme">
+                    <div class="title">
+                    <h1>About me.</h1>
+                    </div>
+
+                        <div class="Obrazek">
+                        <img src="OTSOUE0.png">
+                        </div>
+                        <div class="tekst">
+                        <p>Full name:<span style="color:#f5c842">Designer</span></p>
+                        <p>Age:<span style="color:#f5c842">21</span></p>
+                        <p>Profile:<span style="color:#f5c842">fullstack</span></p>
+                        <p>Email:<span style="color:#f5c842">fullstack@gmail.com</span></p>
+                    </div>
+                    <div class="LoremIpsum">
+                            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+                    </div>
+                </div>
+                <div class="Skils">
+                    <div class="Tytuł">
+                        <h1>Skils</h1>
+                        </div>
+                        <div class="box1">
+                                <img src="Vector (3).png">
+                            <p style="color:#f5c842 ;">UX/UI</p>
```

Comment:
Nie używamy styli inline. Jest to zła praktyka. Możemy nadać odpowiednią klasę z odpowiednimi stylami.